tests/src/data/use-cases/db-add-guardian.spec.ts

```diff
@@ -72,19 +72,19 @@ describe('DbAddGuardian use case', () => {
       await expect(promise).rejects.toThrow()
     })
 
-    it('Should return undefined if email is already registered', async () => {
+    it('Should return null if email is already registered', async () => {
       const { sut, guardianRepositoryStub } = makeSut()
       jest.spyOn(guardianRepositoryStub, 'add').mockResolvedValue(undefined)
       const result = await sut.add(params)
-      expect(result).toBeUndefined()
+      expect(result).toBeFalsy()
```

Comment:
Change to `.toBeNull()`

Reply:
I think that the test name is wrong, because the class DbAddGuardian returns the object added or undefined, soo, I'm not shure but "expect" needs to be undefined. right? Soo, I changed expect to ToBe(undefined) and null in tittle to undefined

Reply:
That's right